### PR TITLE
worked on home-page responsiveness

### PR DESCRIPTION
**Pull request by Enqira**
# Description
worked on responsiveness, also for mobile

Fixes # (issue) No open issue

# How to test?

Run npm run dev and test that hoempage is responsive
# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by qamarfj — CHANGES_REQUESTED** [review]
Great work @Enqira , Its work good on mobiles , But I have noticed at desktop for some screen resolutions buttons are going outside the page.
1280x720
1280x800

File: src/client/containers/Home/Home.styles.css [hunk 1]
```diff
@@ -204,11 +225,31 @@
 }
 
 @media (max-width: 1020px) {
+  header.homepage-header h2 {
+    font-size: 1.5rem !important;
```
Comment (valmdz) [inline src/client/containers/Home/Home.styles.css:229]: Avoid using !important and rather have specific css, do you need` !important `here?

File: src/client/containers/Home/Home.styles.css [hunk 2]
```diff
@@ -264,3 +305,27 @@
...
   }
 }
+
+@media (max-width: 780px) {
+  .description-career {
+    min-height: 340px;
+  }
+
+  .buttons {
+    font-size: 20px !important;
```
Comment (valmdz) [inline src/client/containers/Home/Home.styles.css:315]: same here with the `!important`

Comment (valmdz) [reply src/client/containers/Home/Home.styles.css:315]: Besides that, overall looks very good! So well done, @Enqira

File: src/client/containers/Home/Home.styles.css (hunk 1 above)
Comment (valmdz) [reply src/client/containers/Home/Home.styles.css:229]: We will go with the `!important` this time ;)

**Review by valmdz — APPROVED** [review]

**Review by qamarfj — APPROVED** [review]
great every thing is working it should be.

**Pull request merged**